spec/helper.rb

```diff
@@ -126,10 +133,7 @@ class SingletonClass
     read_ahead_was = Delayed::Worker.read_ahead
     sleep_delay_was = Delayed::Worker.sleep_delay
     min_reserve_interval_was = Delayed::Worker.min_reserve_interval
-
-    if Gem.loaded_specs['delayed'].version >= Gem::Version.new('1.0') && min_reserve_interval_was.zero?
-      raise "Min reserve interval should be nonzero in v1.0 release"
-    end
```

Comment:
Deferring this new default indefinitely, as we have a "duty cycle" feature we're targeting to enable by default instead.